src/index.html

```diff
@@ -70,73 +70,82 @@
+          let scriptElementForZenDesk = document.createElement('script');
+          scriptElementForZenDesk.id = 'ze-snippet';
+          scriptElementForZenDesk.src = 'https://static.zdassets.com/ekr/snippet.js?key=117d80d3-9e4e-4aac-8a1c-1ca9ade4266d';
+          let firstExistingScript = document.getElementsByTagName('script')[0];
+          firstExistingScript.parentNode.insertBefore(scriptElementForZenDesk, firstExistingScript);
+        }, 15000);
 
-      // COMMENT OUT WHEN SPEED TESTING
-      setTimeout(() => {
-        // Delay Stripe load so it doesn't block main thread
-        let firstExistingScript = document.getElementsByTagName('script')[0];
+        setTimeout(() => {
+          // Delay Stripe load so it doesn't block main thread
+          let firstExistingScript = document.getElementsByTagName('script')[0];
 
-        let scriptElement1 = document.createElement('script');
-        scriptElement1.src = 'https://checkout.stripe.com/checkout.js';
-        scriptElement1.type = 'text/javascript';
-        firstExistingScript.parentNode.insertBefore(scriptElement1, firstExistingScript);
+          let scriptElement1 = document.createElement('script');
+          scriptElement1.src = 'https://checkout.stripe.com/checkout.js';
+          scriptElement1.type = 'text/javascript';
+          firstExistingScript.parentNode.insertBefore(scriptElement1, firstExistingScript);
 
-        let scriptElement2 = document.createElement('script');
-        scriptElement2.src = 'https://js.stripe.com/v3/';
-        scriptElement2.type = 'text/javascript';
-        firstExistingScript.parentNode.insertBefore(scriptElement2, firstExistingScript);
-      }, 16000);
+          let scriptElement2 = document.createElement('script');
+          scriptElement2.src = 'https://js.stripe.com/v3/';
+          scriptElement2.type = 'text/javascript';
+          firstExistingScript.parentNode.insertBefore(scriptElement2, firstExistingScript);
+        }, 16000);
+
+        setTimeout(() => {
+          // Maps load so it doesn't block main thread
+          let scriptElementForMaps = document.createElement('script');
+          scriptElementForMaps.src = 'https://maps.googleapis.com/maps/api/js?key=AIzaSyCM0vM1mzh56OuA5IfgW_U6Qd9jD99WH6M&callback=initMap&libraries=places';
+          let firstExistingScript = document.getElementsByTagName('script')[0];
+          firstExistingScript.parentNode.insertBefore(scriptElementForMaps, firstExistingScript);
+        }, 20000);
+      }
 
-      // COMMENT OUT WHEN SPEED TESTING
       setTimeout(() => {
-        // Maps load so it doesn't block main thread
-        let scriptElementForMaps = document.createElement('script');
-        scriptElementForMaps.src = 'https://maps.googleapis.com/maps/api/js?key=AIzaSyCM0vM1mzh56OuA5IfgW_U6Qd9jD99WH6M&callback=initMap&libraries=places';
+        // This is a tiny (28kb) library, and can be needed immediately.  Disabling this load could stop the app from running.
```

Comment:
Hi @SailingSteve, is this comment "This is a tiny (28kb) library" still accurate? If so, maybe we should load it up in the "head" so it never stops the app from running?